data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/beacon/MilestoneDependentTypesUtil.java

```diff
@@ -32,6 +33,8 @@
 
 public class MilestoneDependentTypesUtil {
 
+  private static final Set<SpecMilestone> IGNORED_MILESTONES = Set.of(SpecMilestone.CAPELLA);
```

Comment:
We should make sure we log an issue to unignore this when we're ready. Otherwise we'll forget to make it appear in the api spec for sure...

Reply:
fair point #6355